test/runtests.jl

```diff
@@ -26,7 +26,7 @@ max_worker_rss != typemax(Csize_t) && move_to_node1("parallel")
 cd(dirname(@__FILE__)) do
     n = 1
     if net_on
-        n = min(8, Sys.CPU_CORES, length(tests))
```

Comment:
I don't know. I just made the smallest change. I wonder if there are systems with lots of cores and too little memory. If anything, it would be better to have an environment variable to change this.